apps/rush-lib/src/scripts/install-run.ts

```diff
@@ -433,7 +433,7 @@ export function installAndRun(
     }
   );
 
-  return result.status;
+  return result.status || 1; // If result.status is null, the process was terminated from a signal
```

Comment:
When the executable cannot be found, IIRC the `status` will be undefined and a separate `error` field will have a JavaScript `Error` object with a meaningful message that we should maybe report.